app/controllers/concerns/helper.rb

```diff
@@ -0,0 +1,11 @@
+module Helper
+
+  def logged_in?
+    !!session[:id]
```

Comment:
I assume "memoized" means that it doesn't re-run the method when it's called successively, sort of like a cached value? I'll look into that, good to know! I'll change this too, and make a note for the future--you're totally right, even just in the fact that I should use abstraction whenever possible/appropriate.